src/vs/workbench/contrib/welcomeGettingStarted/browser/gettingStarted.contribution.ts

```diff
@@ -110,10 +110,13 @@ registerAction2(class extends Action2 {
 				}]);
 			} else if (!openedWalkthroughExists) {
 				// else open respecting toSide
-				editorService.openEditor(gettingStartedInput, { preserveFocus: toSide ?? false }, toSide ? SIDE_GROUP : undefined);
+				editorService.openEditor({
```

Comment:
There now seems to be a `const gettingStartedInput =` a few lines up that is not needed unless you are in the `if` branch above?